tests/playwright/.eslintrc.js

```diff
@@ -1,11 +1,13 @@
 module.exports = {
-    extends: '@ni/eslint-config-playwright',
     root: true,
+    ignorePatterns: ['*.js'],
     overrides: [
         {
-            files: [
-                '*.ts',
-            ],
+            extends: '@ni/eslint-config-playwright',
+            files: ['*.ts'],
+            parserOptions: {
+                project: './tsconfig.json'
```

Comment:
> In any case, clients should already be setting parserOptions.project for running eslint on their ts files, so I wouldn't expect this to cause issues.

That was not mandatory before and clients had the option of adopting rules that did not need a tsconfig for type checking (with the trade-off of supporting a smaller rule set). These configurations currently don't require tsconfigs, and the `requiring-type-checking` rulesets are the ones that should require tsconfigs.

I'd like to not couple that change with this PR and not change the behavior of the typescript vs typescript-requiring-type-checking. If that is not possible then we should open the discussion to merging the rulesets since the distinction would no longer be meaningful.

Reply:
It turns out that rule now does in fact require type checking, so I moved it to the other config. I also discovered that there were a lot more changes than I had originally thought (I must not have done a `npm i` in between running `print-evaluated-rules`). I've updated the PR description with detailed information about how I handled those changes.